src/main/java/minegame159/meteorclient/systems/modules/movement/Scaffold.java

```diff
@@ -78,6 +78,20 @@
             .build()
     );
 
+    private final Setting<Boolean> filtFall = sgGeneral.add(new BoolSetting.Builder()
```

Comment:
Perhaps use a clearer setting name. Instead of `filtFall`, use `filterFallingBlocks` or `filterGravityBlocks`